spec/errors.md

```diff
@@ -283,10 +283,10 @@ with a resolved value which does not support selection.
 ## Message Function Errors
 
 A **_<dfn>Message Function Error</dfn>_** is any error that occurs
-when calling a message function implementation
+when calling a _function handler_
 or which depends on validation associated with a specific function.
 
-Implementations SHOULD provide a way for _functions_ to emit 
+Implementations SHOULD provide a way for _function handlers_ to emit 
```

Comment:
I couldn't find a definition of what a "function handler" is. Which file is that in?

Reply:
It's added by this PR to `spec/formatting.md`.

Reply:
Here's a link to the section in this PR's branch: https://github.com/unicode-org/message-format-wg/blob/define-resolved-values/spec/formatting.md#function-handler